### ISSUE TITLE
Visual Studio 2015 Enterprise - No Tests Found.

### ISSUE THREAD

**Issue by davemvp** [mmanela/chutzpah#396]
I am able to run tests using the Context Menu extension just fine but the tests don't show up in Test Explorer even though I have the Test Adapter extension installed.

Is there something special I need to be doing in order for the TestAdapter to "see" my tests?

**Comment by mmanela** [comment]
The Chutzpah adapter tries to run/load all your test files and if this is failing it means that it is picking up someother file as a test file and erroring when trying to create a harness for it. 

Are you sure you are including only the tests you want to run in the chutzpah.json file?

**Comment by davemvp** [comment]
I don't have a chutzpah.json file. I only have referenced the jasminejs nuget package in my unit test project. I did create a simple website with a unit test project. I added a jasmine test there and it showed up. I also noticed for the solution in question that under "Tests" in the Output window there is a message (one or more errors occurred) after the Load Playlist finished entry.

Where would I put the chutzpah.json file?

**Comment by mmanela** [comment]
You put it at the root of your project and use it to explicitly tell chutzpah which are your test files. There is documentation on the wiki here and some samples.

**Comment by davemvp** [comment]
My *.spec.js files are in a separate Unit test Project. I tried to put the Chutzpah.json file in the solution root, the Unit Test Project Root and the website root with no results. Below are the contents of the file.

{
  "Framework": "jasmine",
  "RootReferencePathMode": "SettingsFileDirectory",
  "Tests": [
    {
      "Path": "app",
      "Includes": [ "*.Spec.js", "*.spec.js" ]
    }
  ]
}

**Comment by mmanela** [comment]
Can you create a github repo with a repro of your issue that I can clone and test with?

**Comment by davemvp** [comment]
I created a repository, Test-Chutzpah-Web, where I was able to reproduce the issue.

However, while I preparing to sync the code to GitHub, the extension was updated from Version 4.0.3 to Version 4.1. I restarted Visual Studio and the latest version of the Chutzpah Test Adapter for the Test Explorer (4.1) is discovering the tests for the solution in question with no changes to the configuration. That's great news!

Closing the issue.
